src/submitter.py

```diff
@@ -178,7 +178,7 @@ def extract_crash_id_from_record(record):
     try:
         key = record["s3"]["object"]["key"]
         LOGGER.debug("looking at key: %s", key)
-        if not key.startswith("v2/raw_crash/"):
+        if not key.startswith("v1/raw_crash/"):
             LOGGER.debug("%s: not a raw crash--ignoring", repr(key))
             return None
         crash_id = key.rsplit("/", 3)[-1]
```

Comment:
I have no idea why this is an rsplit for 3 things and then it only looks at the last item, but I lacked the courage and resolve to change it to see what happened.